src/attestation/azext_attestation/generated/_params.py

```diff
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from azext_attestation.action import AddPolicySigningCertificatesKeys
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('attestation list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('attestation show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('provider_name', options_list=['--name', '-n'], help='Name of the attestation service instance',
+                   id_part='name')
+
+    with self.argument_context('attestation create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('provider_name', options_list=['--name', '-n'], help='Name of the attestation service')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx),
+                   validator=get_default_location_from_resource_group,
+                   help='List of available regions for the resource type is \'eastus2,centralus,uksouth\'')
+        c.argument('tags', tags_type)
+        c.argument('attestation_policy', help='Name of attestation policy.')
+        c.argument('policy_signing_certificates_keys', action=AddPolicySigningCertificatesKeys, nargs='+', help='The va'
```

Comment:
Do we want this to be a JSON or file to be read in.? Given the help.py, there is a lot of options here and customers my find it easier to have JSON files for the different crypto algorithms and settings for them they want to use.

Reply:
Good suggestion, JSON would be easier for a complex input. I will update in next commit.